src/uu/stat/src/stat.rs

```diff
@@ -501,7 +501,7 @@ impl Stater {
             // mount points aren't displayed when showing filesystem information
             None
         } else {
-            let mut mount_list = read_fs_list()
+            let mut mount_list = read_fs_list()?
```

Comment:
I'm wondering about this case, because it's currently missing a context. Do we know that GNU's error in this case is?

Reply:
I tested the Rust and GNU versions of stat in the same environment I used to test df (chroot without proc).
While the Rust version fails, the GNU version does not but displays the expected output. For this reason I was not able to produce the error with GNU's stat.

I could only find GNU's error message in the sources:

https://github.com/coreutils/coreutils/blob/93e099e4c3b659b2e329f655fbdc73fdf594a66e/src/stat.c#L961